rfcs/0145-doc-strings.md

```diff
@@ -0,0 +1,501 @@
+---
+feature: doc-comment-standard
+start-date: 2023-03-27
+author: hsjobeki
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (@DavHau; tbd.)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Propose a standard format specification for Doc-comments.
+
+This RFC includes two concerns that define a doc-comment:
+
+- Outer format rules to allow distinction between regular comments and doc-comments
+- Inner format rules that describe the required format of the content. 
+
+However, both concerns relate closely to each other; It makes sense and reduces bureaucracy to address that in a single RFC.
+
+# Motivation
+[motivation]: #motivation
+
+The following are the envisioned goals.
+
+- Be able to generate documentation from code for any nix expression.
+- Be able to distinguish between documentation-relevant comments and unrelated comments.
+- Make doc comments easy to write and read
+- Be able to parse and render doc comments nicely
+- Standardize a format for doc comments that further RFCs can extend
+
+This RFC is a significant change to existing documentation conventions.
+It allows distinguishing between regular and doc comments. Having distinction is essential because arbitrary code comments should not end up in generated documentation.
+
+> Hint: Generating static documentation is controvert topic in nixpkgs. We found that it is impossible to generate accurate documentation statically. A correct solution would involve the evaluation of expressions in some way.
+
+## Current State
+
+Currently, Nix does not specify doc comments. Over time there evolved certain conventions, But the nix-community never specified those.
```

Comment:
```suggestion
Currently, Nix does not have doc comments. Over time there evolved certain conventions, but they were never formally specified.
```

Nix-Community is a sibling organization. Better avoid confusion.

Also, remove some redundancy.